src/UI/toolbar/FlyViewToolBar.qml

```diff
@@ -153,6 +154,24 @@ Rectangle {
         }
     }
 
+    //-- FoxFour Button
+    Image{
+        id:foxFourLogo
+        anchors.right:          parent.right
+        anchors.top:            parent.top
+        anchors.bottom:         parent.bottom
+        anchors.margins:        5
+        source:                 "qrc:/qmlimages/F4/FoxFourTextLogo.svg"
+        width:                  _activeVehicle && _currentComputer? 70:0
```

Comment:
Please, use `visible` property to hide the element. It is much more clear what condition makes the element to disappear